test/UglifyJsPlugin.test.js

```diff
@@ -122,7 +122,7 @@ describe("UglifyJsPlugin", function() {
 						}], function() {
 							compilation.errors.length.should.be.exactly(1);
 							compilation.errors[0].should.be.an.Error;
-							compilation.errors[0].message.should.have.containEql("SyntaxError");
+							compilation.errors[0].message.should.have.containEql("Unexpected token");
```

Comment:
The full `.stack` string hasn't, no. But the `SyntaxError` part now comes from the `Error`'s type instead of hardcoded as part of `message`.